hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFilePreadReader.java

```diff
@@ -150,8 +153,8 @@ public void run() {
             }
           } catch (IOException e) {
             // IOExceptions are probably due to region closes (relocation, etc.)
-            if (LOG.isTraceEnabled()) {
-              LOG.trace("Prefetch " + getPathOffsetEndStr(path, offset, end), e);
+            if (LOG.isDebugEnabled()) {
+              LOG.debug("Prefetch " + getPathOffsetEndStr(path, offset, end), e);
```

Comment:
Would be nice to move to parameterized logging.

Reply:
problem is that we are also logging the Throwable.